app/models/user.rb

```diff
@@ -1,17 +1,32 @@
 class User < ActiveRecord::Base
+  ROLES = %w[superadmin admin volunteer]
+
   TEMP_EMAIL_PREFIX = 'change@me'
   TEMP_EMAIL_REGEX = /\Achange@me/
 
   devise :database_authenticatable, :registerable, :recoverable,
          :rememberable, :trackable, :validatable, :omniauthable
 
-  # Setup accessible (or protected) attributes for your model:
-  attr_accessible :email, :password, :password_confirmation, :remember_me
+  attr_accessible :email, :password, :password_confirmation, :remember_me, :role
 
   validates_format_of :email, :without => TEMP_EMAIL_REGEX, on: :update
 
+  after_create :assign_default_role
+
+  def is?(role)
+    role.to_s == self.role
+  end
+
+  def assign_default_role
+    self.role ||= 'volunteer'
+    self.save
+  end
```

Comment:
I would prefer not to hardcode it in the database because this is subject to change. Adding it in migrations makes it moderately permanent, and changing that would require another migration.